components/image/image.css

```diff
@@ -0,0 +1,17 @@
+.responsive-image {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
```

Comment:
The image is not responsive if the height of the img-container increases. Need to work on this.

Reply:
Done